docs/yul.rst

```diff
@@ -1015,3 +1015,54 @@ If you want to use Solidity in stand-alone Yul mode, you activate the optimizer
     solc --strict-assembly --optimize
 
 In Solidity mode, the Yul optimizer is activated together with the regular optimizer.
+
+Optimization step sequence
+--------------------------
+
+By default Yul optimizer applies its predefined sequence of optimization steps to the generated assembly.
+You can override this sequence and supply your own using the `--yul-optimizations` option when compiling in Solidity mode:
+
+.. code-block:: sh
+
+    solc --optimize --ir-optimized --yul-optimizations dhfoD[xarrscLMcCTU]uljmul
```

Comment:
```suggestion
    solc --optimize --ir-optimized --yul-optimizations 'dhfoD[xarrscLMcCTU]uljmul'
```
(just in case the shell does weird things with the `[]`

Reply:
Well, I switched to `[]` specifically to avoid shell doing weird things to `()`. But ok, I'll change it. You can never be too careful with shell :)

Reply:
Done.